Objects/longobject.c

```diff
@@ -415,6 +415,18 @@ PyLong_FromUnsignedLong(unsigned long ival)
 PyObject *
 PyLong_FromDouble(double dval)
 {
+    /* Try to get out cheap if this fits in a long.  The attempt
```

Comment:
Might make sense to use `long long` or `intmax_t` here instead of `long`, so this path will be used on ARM32.

Reply:
Can you guarantee that operations with `long long` are as effective as with `long` and that this will not add an overhead for small values?

Reply:
No, so I'm resolving this.